modules/stream_analytics/main.tf

```diff
@@ -13,20 +13,18 @@ provider "azurerm" {
 }
 
 locals {
-  stream_output_name = "${var.name}-output"
-  stream_input_name  = "${var.name}-input"
-  stream_job_name    = "${var.name}-job"
-
-  select_all_query = "SELECT * INTO \"${local.stream_output_name}\" FROM \"${local.stream_input_name}\""
```

Comment:
Maybe we still need to have the default query here?

Reply:
The default query can not longer be used since there can now be multiple in- and outputs

Reply:
I understand, any other options for a default query? Otherwise, leave it empty.